journal-letters/modernized/10-chapter-9.adoc

```diff
@@ -43,23 +45,32 @@ which he had cause to regret, and remarked:
 "`I hope experience will teach greater degrees of obedience.`"
 When they had safely come near land, he wrote:]
 
-
 I trust I am thankful for the Lord's mercies and favours daily dispensed,
 and can say in humility, I have not murmured since my coming on board;
 but have been mercifully preserved in a sense of His goodness.
 I hope for this blessing to be continued, that according to ability,
 I may labour in the vineyard,
 and work the works of Him who has called me with his high and holy calling.
 
+[.small-break]
+'''
```

Comment:
maybe offset here too? or possibly i'm missing a difficulty, i know you said on Sarah Grubb you had some reasoning for lots of small-breaks, maybe we should just chat about these.